cassdegrees/ui/views/student.py

```diff
@@ -100,6 +101,23 @@ def student_create(request):
         response = redirect(student_edit)
         response['Location'] += '?plan='
         return response
+    # If a plan name is given, copy that plan
+    elif plan_to_duplicate:
+        plan = decompress(request.session.get('plan:' + plan_to_duplicate, None))
+        print(plan)
+        if plan:
+            new_plan_name = plan['name'] + ' (Copy)'
+            plan['name'] = new_plan_name
+            plan['date'] = timezone.localtime().strftime('%d/%m/%Y %H:%M')
```

Comment:
Are we still using localtime? Did we come to a consensus on this?

Reply:
We needed to use it as otherwise we would be giving the wrong time to the users. This instance is a purely front-facing feature so using localtime isn't a huge issue